vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -1,19 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-// clang-format off
```

Comment:
Love this change, but for the future, it would be nice to put something like this in its own PR. It helps reviewers pick out what has changed, but also can help reduce the scope of the review to just what has changed vs folks maybe looking at more parts of this code that are out of scope.

Reply:
Noted.

Will apply this criteria for following PRs.